Core/AppRuntime/Source/WorkQueue.cpp

```diff
@@ -42,6 +43,15 @@ namespace Babylon
         while (!m_cancelSource.cancelled())
         {
             m_dispatcher.blocking_tick(m_cancelSource);
+
+            // Call the unhandled exception handler if an exception is thrown in one of the tasks.
+            m_task = m_task.then(arcana::inline_scheduler, arcana::cancellation::none(), [this](const arcana::expected<void, std::exception_ptr>& result) {
+                if (result.has_error())
+                {
+                    m_unhandledExceptionHandler(result.error());
+                    m_cancelSource.cancel();
```

Comment:
Are you sure we want to stop the loop? It seems like it is a lot more typical in JS environments to continue execution.

Reply:
Huh? JS shouldn't continue if there is an unhandled exception?

Reply:
Removed.